src/modules/Modal/Modal.js

```diff
@@ -373,12 +373,12 @@ Modal.propTypes = {
   /** Custom styles. */
   style: PropTypes.object,
 
-  /** Element to be rendered in-place where the portal is defined. */
+  /** Element to be rendered in-place where the modal is defined. */
   trigger: PropTypes.node,
 
   /**
-   * NOTE: Any unhandled props that are defined in Portal are passed-through
-   * to the wrapping Portal.
+   * NOTE: Any unhandled props that are defined in Modal are passed-through
+   * to the wrapping Modal.
```

Comment:
```suggestion
   * to the inner Portal.
```

This is more accurate 😉

Reply:
Of course :)